### DISCUSSION TITLE
mask plugin doesnt work in Docker

### DISCUSSION THREAD

**Discussion by caueassmann** [alpinejs/alpine#3623] [1. Help]
Is there any configuration needed to the mask plugin specifically works with Docker + Laravel?

I created a docker-compose with 3 DockerFiles: nginx, php and node. 

This is the node DockerFile:

`
FROM node:16-alpine

WORKDIR /var/www/html

EXPOSE 5173

CMD ["sh", "-c", "npm install && npm run build"]

`
In app.js:
import './bootstrap';

import Alpine from 'alpinejs';
import focus from '@alpinejs/focus';
import mask from '@alpinejs/mask'
window.Alpine = Alpine;

Alpine.plugin(focus,mask);

Alpine.start();

I dont know why this happens, but i receive this message: "Alpine Warning: You can't use [x-mask] without first installing the "Mask" plugin here"

I tried to execute npm install @alpinejs/mask  inside the node container but with no effect

**Comment by caueassmann** [comment]
The error was in the .env file, the url was incorrect.

**Comment by ekwoka** [comment]
The syntax for Alpine.plugin is wrong.

It was nothing to do with docker.

`Alpine.plugin` accepts ONE plugin or ONE array of multiple plugins.
